packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -131,6 +132,42 @@ export abstract class BucketRef extends cdk.Construct {
     });
   }
 
+  /**
+   * Defines a CloudWatch Event Rule that triggers upon putting an object into the Bucket.
+   *
+   * @param name the logical ID of the newly created Event Rule
+   * @param target the target of the Event Rule
+   * @param path the optional path inside the Bucket that will be watched for changes
+   * @returns a new {@link events.EventRule} instance
+   */
+  public onObjectPut(name: string, target?: events.IEventRuleTarget, path?: string): events.EventRule {
```

Comment:
Please rename this to onPutObject to align with the cloudwatch event name

Reply:
Done.